patches/master_patch.patch

```diff
@@ -1284,6 +1284,28 @@ index 2b38db266f9aa1f4141c8649c021042ede4e5589..ea0387b11f4b00f87596d738c152aa4c
    OSStatus error = keychain_.FindGenericPassword(
        nullptr, strlen(service_name), service_name, strlen(account_name),
        account_name, &password_length, &password_data, NULL);
+diff --git a/components/ssl_config/ssl_config_service_manager_pref.cc b/components/ssl_config/ssl_config_service_manager_pref.cc
+index c2e85271796d6dbc510cd6c5682fc98a2ea194a1..e5b776b672327feafbd72fbe5e80291b9589bb55 100644
+--- a/components/ssl_config/ssl_config_service_manager_pref.cc
++++ b/components/ssl_config/ssl_config_service_manager_pref.cc
+@@ -308,10 +308,13 @@ void SSLConfigServiceManagerPref::GetSSLConfigFromPrefs(
+     net::SSLConfig* config) {
+   // rev_checking_enabled was formerly a user-settable preference, but now
+   // it is managed-only.
+-  if (rev_checking_enabled_.IsManaged())
+-    config->rev_checking_enabled = rev_checking_enabled_.GetValue();
+-  else
+-    config->rev_checking_enabled = false;
++  #if defined(MUON_CHROMIUM_BUILD)
++    if (rev_checking_enabled_.IsManaged())
```

Comment:
don't indent here because it makes the diff harder to read. preprocessor are never indented anyway so that will effectively indent code block without adding any spaces